File: app/src/main/java/com/cere/csplayer/until/DialogUtils.java
package com.cere.csplayer.until;

import android.app.AlertDialog;
import android.content.Context;

import androidx.annotation.DrawableRes;

/**
 * Created by CheRevir on 2019/10/24
 */
public class DialogUtils {
    public static AlertDialog.Builder getAlertDialog(Context context, String title, String message, @DrawableRes int icon) {
        AlertDialog.Builder builder = new AlertDialog.Builder(context);
        builder.setTitle(title);
        builder.setMessage(message);
        builder.setIcon(icon);
        return builder;
    }
}

File: app/src/main/java/com/cere/csplayer/util/DialogUtils.kt
package com.cere.csplayer.util

import android.app.AlertDialog
import android.content.Context
import androidx.annotation.DrawableRes

/**
 * Created by CheRevir on 2019/10/24
 */
object DialogUtils {
    @JvmStatic
    fun getAlertDialog(context: Context, title: String, message: String, @DrawableRes icon: Int): AlertDialog.Builder {
        val builder = AlertDialog.Builder(context)
        builder.setTitle(title)
        builder.setMessage(message)
        builder.setIcon(icon)
        return builder
    }
}